about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css' rel='stylesheet'>
+  <link rel="stylesheet" href="about.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="show-case">
+    <div class="overlay"></div>
+    <header>
+      <div class="social">
+        <ul class="social-links d-flex">
+          <li><a href="#"><i class='bx bxl-facebook-circle bx-sm'></i></a></li>
+          <li><a href="#"><i class='bx bxl-twitter bx-sm'></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <nav class="d-flex">
+        <a class="burger" href="#">
+          <i class='bx bx-menu bx-md'></i>
+        </a>
+        <a href="index.html">
+          <h1 class="frank-logo" >Musica</h1>
+        </a>
+        <ul class="desktop-links">
+          <li class="primary-color"><a href="about.html">About</a></li>
```

Comment:
- [ ] The about page nav should contain a home link page.

![nav](https://user-images.githubusercontent.com/61562626/142411953-a7a5151d-d9b9-4a48-a7c7-1ad2431fed55.jpeg)

Reply:
Addressed this